tslint.json

```diff
@@ -33,7 +33,7 @@
             150
         ], 
         "no-angle-bracket-type-assertion": false,
-        "no-implicit-dependencies": [true, "dev"],
+        "no-implicit-dependencies": false,

```

Comment:
Why change this?

Reply:
Updated to whitelist vscode module for tslint